google/cloud/forseti/scanner/scanners/iap_scanner.py

```diff
@@ -272,7 +272,7 @@ def make_iap_resource(self, backend_service, project_full_name):
         for backend in backend_service.backends:
             instance_group = self.find_instance_group_by_url(
                 backend.get('group'))
-            if not instance_group:
+            if not instance_group or not backend_service.port:
```

Comment:
Are we still able to scan the resources correctly, if we suppress the missing port here?

i.e. will IapResource still be constructed correctly, and then evaluated later for violation?

Reply:
Updated with detail comment.